agent/one_prompter.py

```diff
@@ -0,0 +1,190 @@
+"""An LLM agent to generate a simple fuzz target prototype that can build.
```

Comment:
missing license?

Reply:
Yep let me add this.
Thanks!

Reply:
Oh I just noticed that so many files are missing license headers.
The corresponding CIs must have been broken for a long time.
I will fix that in later PRs.